docs/LayoutWithFlexbox.md

```diff
@@ -1,10 +1,10 @@
 ---
-id: basics-layout
+id: layout-with-flexbox
 title: Layout with Flexbox
 layout: docs
 category: The Basics
 permalink: docs/flexbox.html
```

Comment:
Oops! I guess the id needs to match the permalink. We could really use docs infra with a saner permalink / redirect system. cc @bestander :D